indexing-service/src/main/java/org/apache/druid/indexing/common/task/NoopTask.java

```diff
@@ -67,18 +62,14 @@ enum IsReadyResult
   @JsonIgnore
   private final IsReadyResult isReadyResult;
 
-  @JsonIgnore
-  private final FirehoseFactory firehoseFactory;
-
   @JsonCreator
   public NoopTask(
       @JsonProperty("id") String id,
       @JsonProperty("groupId") String groupId,
       @JsonProperty("dataSource") String dataSource,
-      @JsonProperty("runTime") long runTime,
+      @JsonProperty("runTime") long runTimeMillis,
       @JsonProperty("isReadyTime") long isReadyTime,
       @JsonProperty("isReadyResult") String isReadyResult,
```

Comment:
Is a non-null value being passed for `isReadyResult`?

Reply:
Not really. I guess we should remove this too.